src/main/java/uk/gov/hmcts/ccd/domain/model/search/elasticsearch/ElasticsearchRequest.java

```diff
@@ -88,4 +164,9 @@ private String getFieldId(String fieldSourceName) {
             }
         }
     }
+
+    private boolean arrayContainsOnlyText(ArrayNode node) {
+        return StreamSupport.stream(node.spliterator(), false)
+            .allMatch(JsonNode::isTextual);
+    }
```

Comment:
this is part of the request validation, so it would be better here probably? 
https://github.com/hmcts/ccd-data-store-api/blob/17f3cbe287ac5de485d0c9a875339dcc908bc40d/src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/ElasticsearchQueryHelper.java#L31